internal/buildscripts/packaging/installer/install.sh

```diff
@@ -871,12 +871,11 @@ Auto Instrumentation:
                                         applications running as systemd services.
                                         Cannot be combined with the '--with-instrumentation' option.
                                         (default: --without-systemd-instrumentation)
-  --with[out]-java-instrumentation      Whether to enable Auto Instrumentation for Java.
-                                        (default: --with-java-instrumentation if --with-instrumentation or
+  --with[out]-instrumentation-sdk "<s>" Whether to enable Auto Instrumentation for a specific language. This option
```

Comment:
Instead of a comma-separated list of languages, would it be simpler to just allow specifying the options multiple times for each language, e.g. `--with[out]-instrumentation-sdk node --with[out]-instrumentation-sdk java`?

Reply:
No, @aunshc didn't want this